Index: Step 0 - Delete salto.remove
===================================================================
--- Step 0 - Delete salto.remove
+++ Step 0 - Delete salto.remove
@@ -1,4 +1,0 @@
-type salto_remove {
-  string before {
-  }
-}
\ No newline at end of file

Index: Step 1 - Modify salto.modify
===================================================================
--- Step 1 - Modify salto.modify
+++ Step 1 - Modify salto.modify
@@ -1,4 +1,4 @@
 type salto_modify {
-  string before {
+  string after {
   }
 }

Index: Step 2 - Add salto.add
===================================================================
--- Step 2 - Add salto.add
+++ Step 2 - Add salto.add
@@ -1,0 +1,4 @@
\ No newline at end of file
+type salto_add {
+  string before {
+  }
+}
